src/db/core.ts

```diff
@@ -400,19 +411,31 @@ export class SQLFragment<RunResult = pg.QueryResult['rows'], Constraint = never>
       const columnNames = <Column[]>Object.keys(expression).sort();
```

Comment:
This block could be simplified if the columns are filtered here. Then there is no need for `firstField`

```
const columnNames = <Column[]>Object.keys(expression)
  .filter(key => (<any>expression)[key] !== undefined)
  .sort();
```